recipes/canfestival/canfestival.oe

```diff
@@ -0,0 +1,55 @@
+DESCRIPTION = "canfestival"
+LICENSE = "LGPLv2.1"
+
+inherit make
+SRC_REV = "8bfe0ac00cdb"
+SRC_URI = "http://dev.automforge.net/CanFestival-3/archive/${SRC_REV}.tar.gz \
+	file://0001-install-pkgconfig-module-for-canfestival.patch \
+	file://0002-allow-to-set-python-interpreter.patch"
+
+S = "${SRCDIR}/CanFestival-3-${SRC_REV}"
+
+DEPENDS = "libc libpthread librt libdl"
+RDEPENDS = "libc libpthread librt libdl"
+
+do_configure() {
+	${S}/configure \
+		--target=unix \
+		--arch=arm \
+		--timers=unix \
+		--cc="${HOST_ARCH}-gcc" \
+		--cxx="${HOST_ARCH}-g++" \
+		--ld="${HOST_ARCH}-gcc" \
+		--prefix=/usr \
+		--can=socket
+}
+
+do_compile() {
+	oe_runmake -C ${S} all \
+		PYTHON=/usr/bin/python2
```

Comment:
missing python depends, also use the major version logic in python. not sure what we do when we start introducing a dependency on meta/python here ... @esben  ?